spec/fundamentals.mediawiki

```diff
@@ -7,15 +7,37 @@ peer-to-peer trades with the mechanics of order execution existing entirely on
 the separate blockchains via atomic swaps.
 These are:
 * Asset-specific order quantity increments and transaction fee rates
-* Epoch-based pseudorandom order matching
+* Epoch-based pseudorandom order matching, with epoch duration configure per-market
 * Client identities based on public key infrastructure (PKI)
 * An open and rigidly-defined interface for integration of arbitrary assets
 
 This section describes each of these design aspects.
 
 ==Exchange Variables==
 
-There are a number of asset-specific variables that must be known by the client.
+There are a number of exchange-wide, market-specific, and asset-specific
+variables that must be known by the client. These settings should be requested
+by the user immediately after connecting via the
+[[#Configuration_Data_Request|<code>config</code> request]].
+
+===Global Variables===
+
+The cancellation ratio for a user is the ratio of their canceled order count to
+completed order count. A user's cancellation ratio must remain below the configured
+[[community.mediawiki/#Rule_3_An_accounts_cancellation_ratio_must_remain_below_the_threshold|'''cancellation threshold''']]
+(<code>cancelmax</code>) to avoid penalty.
+
+The '''broadcast timeout''' (<code>btimeout</code>) is the amount of time a
+client has to broadcast a transaction. For the maker, the broadcast time is
+measured from the time of match notification. For the taker, timeout is measured
+from the time when the maker's swap receives its
+<code>swapconf</code><sup>th</sup> confirmation (see [[#Asset_Variables|Asset Variables]]).
+
+The user's transaction that pays the registration fee specified by the
+<code>register</code> response during [[accounts.mediawiki/#Account_Creation|account creation]]
+must have <code>regfeeconfirms</code> confirmations to be considered paid.
```

Comment:
It actually must have `regfeeconfirms` before the server will even accept the `notifyfee`, though the distinction isn't too important here.